components/admin_console/team_channel_settings/channel/details/channel_details.tsx

```diff
@@ -153,51 +153,49 @@ export default class ChannelDetails extends React.PureComponent<ChannelDetailsPr
         }
     }
 
-    async componentDidMount() {
+    componentDidMount() {
```

Comment:
React doesn't wait on an async `componentDidMount` and neither did our tests, so I removed the async/await for simplicity.